app/build.gradle

```
@@ -49,6 +50,21 @@ dependencies {
 
     def coil_version = "1.4.0"
     implementation "io.coil-kt:coil:$coil_version"
+    def room_version = "2.4.1"
+    implementation "androidx.room:room-ktx:$room_version"
+    kapt "androidx.room:room-compiler:$room_version"
+    annotationProcessor "androidx.room:room-compiler:$room_version"
```

используется только kapt, это по сути одно и тоже, kapt - новый метод
```suggestion
    kapt "androidx.room:room-compiler:$room_version"
```